go.mod

```diff
@@ -13,6 +13,7 @@ replace github.com/docker/docker => github.com/docker/engine v1.4.2-0.2019101121
 
 // Old version had no license
 replace github.com/chzyer/logex => github.com/chzyer/logex v1.1.11-0.20170329064859-445be9e134b2
+replace istio.io/api => github.com/hanahmily/api v0.0.0-20200422080657-e387a4b4939a
```

Comment:
Once https://github.com/tetrateio/api/pull/28/ is merged, delete this line.

Reply:
LGTM once this changes